grpc/server.go

```diff
@@ -43,8 +44,27 @@ func (s *Server) StreamSearch(req *v1.SearchRequest, ss v1.WebserverService_Stre
 		return status.Error(codes.InvalidArgument, err.Error())
 	}
 
-	onMatch := stream.SenderFunc(func(res *zoekt.SearchResult) {
-		ss.Send(res.ToProto())
+	onMatch := stream.SenderFunc(func(r *zoekt.SearchResult) {
+		result := r.ToProto()
+
+		f := func(filesChunk []*v1.FileMatch) error {
+			return ss.Send(&v1.SearchResponse{
+				Stats:         result.GetStats(),
+				Progress:      result.GetProgress(),
+				RepoUrls:      result.GetRepoUrls(),
```

Comment:
Note: Is there ever a case where there are entries in RepoURLs that _aren't_ one of the repositories mentioned in the file matches?